src/non-standalone/Dialog/ImageDialog.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import {
+	Dialog,
+	DialogContent,
+	makeStyles,
+	IconButton,
+} from "@material-ui/core";
+import MuiDialogTitle from "@material-ui/core/DialogTitle";
+import { Close } from "@material-ui/icons";
+import { useDialogContext } from "../../framework";
+import {
+	ImageController,
+	IDialogConfigImageBox,
+} from "../../standalone/ImageBoxControl/index";
+export interface ImageBoxDialogProps
+	extends Omit<IDialogConfigImageBox, "classes"> {
+	/**
+	 * Custom styles
+	 */
+	classes?: Partial<ReturnType<typeof useStyles>>;
```

Comment:
How can I style the image controller? Add
```tsx
subClasses?: {
  imageController?: IDialogConfigImageBox["classes"];
}
```
and pass the property to the image controller